### PR TITLE
Add image b6

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Profile can now be updated
*You can add image to post, and change profile picture, profile background picture
*You can use emoji
*Logout added

### CODE REVIEW

File: src/components/modals/edit-profile-modal.jsx [hunk 1]
```diff
@@ -1,48 +1,96 @@
...
-  const showModal = () => {
-    setEditProfileModal(true);
-  };
+  const [inputFields, setInputFields] = useState(userData?.userData);
+  const [isLoading, setIsLoading] = useState(false);
 
-  const handleOk = () => {
+  const handleOk = async () => {
+    setIsLoading(true);
+    console.log(inputFields);
```
Comment (saadahmadhere) [inline src/components/modals/edit-profile-modal.jsx]: remove console.log

Comment (AnuragSahu11) [reply src/components/modals/edit-profile-modal.jsx]: yup

File: src/components/modals/create-post-modal.jsx [hunk 2]
```diff
@@ -14,6 +16,20 @@ const NewPostModal = () => {
...
+    setShowEmojiPicker((prevState) => !prevState);
+  };
+
+  const onEmojiClick = (event, { emoji }) => {
+    console.log(emoji);
+    setInputField((prevState) => ({
+      ...inputField,
+      content: prevState.content + emoji,
+    }));
+    console.log(inputField);
```
Comment (saadahmadhere) [inline src/components/modals/create-post-modal.jsx]: remove console.log

**Review by saadahmadhere — APPROVED** [review]
- Nice and modular folder structure
- Code has good readability.
- good to merge.

**Review by rishichourasia — APPROVED** [review]
Awesome work done on the modals . Code can be refactored into smaller components , otherwise all things work perfectly fine. Good to merge

File: src/components/modals/create-post-modal.jsx (hunk 2 above)
Comment (AnuragSahu11) [reply src/components/modals/create-post-modal.jsx]: yup

**Pull request merged**